src/test/java/org/folio/rest/impl/InvoicesProratedAdjustmentsTest.java

```diff
@@ -123,7 +123,7 @@ public void testUpdateInvoiceWithOneLineAddingAdjustment(Adjustment.Prorate pror
      * "Amount" - this is the same as adjustment value
      * "Percentage" - this is the percentage of subTotal i.e. 15% of 25$ = 3.75$
      */
-    double expectedAdjTotal = (type == AMOUNT) ? 25d : 13.75d;
+    double expectedAdjTotal = (type == AMOUNT) ? 65d : 31.25d;
```

Comment:
Invoice has prorated adjustment with value 15. Line has not prorated amount adjustment with value 10$ and subTotal 25$.
When prorated adjustment type is amount `expectedAdjTotal` should be  - 10.0 (line not prorated) + 15.0 (invoice prorated) = 25.
If prorated adjustment type is percentage `expectedAdjTotal` should be  - 10.0 (line not prorated) + 15 * 25 / 100 = 13.75